authenticator_sdk/src/main/kotlin/com/saltedge/authenticator/sdk/AuthenticatorApiManager.kt

```diff
@@ -165,4 +169,14 @@ object AuthenticatorApiManager : AuthenticatorApiManagerAbs {
                 )
             )
     }
+
+    /**
+     * Initialize SDK
+     *
+     * @param userAgent contain up-to-date information about the Application and the user's device
+     */
+    override fun initializeSDK(context: Context): String {
+        userAgentInfo = buildUserAgent(context)
+        return userAgentInfo
```

Comment:
do you have any consumer of this info outside SDK. if not, then `return is redundant.